packages/global/components/layouts/content/webinar.marko

```diff
@@ -0,0 +1,168 @@
+    <div class="content-page-header">
+      <theme-content-page-breadcrumbs section=primarySection />
+      <marko-web-content-name tag="h1" block-name=blockName obj=content />
+      <if(authors.length === 1)>
+        <theme-author-published-node author=authors[0] content=content lazyload=false />
+      </if>
+      <else-if(authors.length > 1)>
+        <default-theme-content-attribution obj=content elements=["authors"] />
+      </else-if>
+      <if(authors.length !== 1)>
+        <default-theme-page-dates|{ blockName }|>
+          <theme-content-published-node block-name=blockName obj=content />
+        </default-theme-page-dates>
+      </if>
+      <global-sponsored-section-logo alias=primarySection.alias modifiers=["content-page"] class="mt-3" />
+        <marko-web-social-sharing
+          path=content.siteContext.path
+          providers=["print", "facebook", "linkedin", "twitter", "pinterest"]
+          print-path=`/print/content/${content.id}`
+        />
+    </div>
+  </@section>
+
+  <@section|{ aliases, content, authors }| modifiers=["content-body"]>
+    $ const { primarySection } = content;
+    <div class="content-page-body">
+      <default-theme-page-contents|{ blockName }| attrs={ "data-gallery-id": id }>
+        $ const requiresRegistration = (get(content, "userRegistration.isCurrentlyRequired") || contentGatingHandler({ content, req }));
+        $ const accessLevels = getAsArray(content, "userRegistration.accessLevels");
+        <marko-web-identity-x-access|context|
+          enabled=requiresRegistration
+          required-access-level-ids=accessLevels
+        >
+          <if(!context.canAccess || context.requiresUserInput)>
+            $ const body = getContentPreview({ body: content.body, selector: "p:lt(3)" });
+            <marko-web-content-body block-name=blockName obj={ body } />
+
+            <div class="content-page-preview-overlay" />
+            <theme-content-page-gate
+              can-access=context.canAccess
+              is-logged-in=context.isLoggedIn
+              has-required-access-level=context.hasRequiredAccessLevel
+              requires-access-level=context.requiresAccessLevel
+              requires-user-input=context.requiresUserInput
+              messages=context.messages
+            />
+          </if>
+          <else>
+
+            $ const bodyId = `content-body-${content.id}`;
+
+            <marko-web-content-body block-name=blockName obj=content attrs={ id: bodyId } />
+
+            <!-- needs input -->
+            <if(input.afterBody)>
+              <${input.afterBody}
+                aliases=aliases
+                block-name=blockName
+                content=content
+              />
+            </if>
+
+            $ const sidebars = getAsArray(content, "sidebars").map((sidebar) => sidebar.body);
+            <marko-web-content-sidebars block-name=blockName obj={ sidebars } />
+
+            <if(leadersContentTypes.includes(content.type))>
```

Comment:
This if check can be removed along with the one on line 144.  Please also remove the const on line 9.